cliquet/tests/support.py

```diff
@@ -71,17 +71,22 @@ class BaseWebTest(object):
     """
 
     api_prefix = "v0"
+    authorization_policy = 'cliquet.tests.support.AllowAuthorizationPolicy'
+    collection_url = '/mushrooms'
```

Comment:
While it's easier to use, I'm not  sure we should put a collection url by default. (explicit vs implicit)